RecoLocalCalo/EcalRecProducers/python/ecalUncalibRecHitPhase2_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+ecalUncalibRecHitPhase2 = cms.EDProducer('EcalUncalibRecHitPhase2WeightsProducer',
+    EBhitCollection = cms.string('EcalUncalibRecHitsEB'),
+    tRise = cms.double(0.2),
+    tFall = cms.double(2.),
+    BarrelDigis = cms.InputTag("simEcalUnsuppressedDigis",""),
+    # weights calculated averaging the full EB
+    weights = cms.vdouble( -0.121016, -0.119899, -0.120923, -0.0848959, 0.261041, 0.509881, 0.373591, 0.134899, -0.0233605, -0.0913195, -0.112452, -0.118596, -0,121737, -0,121737, -0,121737, -0,121737 )
+)
```

Comment:
```suggestion
import  RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2WeightsProducer_cfi as _mod
ecalUncalibRecHitPhase2 = _mod.ecalUncalibRecHitPhase2WeightsProducer.clone()
```
it looks like the values defined in the `fillDescriptions` is the same as what's in this earlier defined file.